### PR TITLE
Add build RedOS linux package

### PR DESCRIPTION
**Pull request by foxzi**
Add the redos_7.3.yml workflow to build Tarantool packages (x86_64) for
the RedOS 7.3 system.

Packages are created by https://github.com/packpack/packpack.

Closes tarantool/tarantool-qa#260

### CODE REVIEW

File: .github/workflows/redos_7_3.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+name: redos_7_3
+
+on:
+  push:
+    branches:
+      - 'master'
+      - '[0-9].[0-9]+'
+      - 'foxzi-build-redos-rpm'
```
Comment (ylobankov) [inline .github/workflows/redos_7_3.yaml:8]: Please delete this branch.

**Review by ylobankov — CHANGES_REQUESTED** [review]
Please add a changelog entry with this commit like it is done here https://github.com/tarantool/tarantool/commit/fccb6bc0870451152937ddf535a953877fe1fe68.

File: .github/workflows/redos_7_3.yaml (hunk 1 above)
Comment (foxzi) [reply .github/workflows/redos_7_3.yaml]: done

**Review by ylobankov — CHANGES_REQUESTED** [review]
Please add a changelog entry with this commit like it is done here https://github.com/tarantool/tarantool/commit/fccb6bc0870451152937ddf535a953877fe1fe68.

**Comment by foxzi** [comment]
> 

done

**Review by ylobankov — COMMENTED** [review]
The patch looks almost good. Could you please rewrite the commit message to something like this?

```
ci: add RedOS 7.3 rpm package build (x86_64)

Add the redos_7.3.yml workflow to build Tarantool packages (x86_64) for
the RedOS 7.3 system.

Packages are created by https://github.com/packpack/packpack.
```

**Comment by foxzi** [comment]
> The patch looks almost good. Could you please rewrite the commit message to something like this?
> 
> ```
> ci: add RedOS 7.3 rpm package build (x86_64)
> 
> Add the redos_7.3.yml workflow to build Tarantool packages (x86_64) for
> the RedOS 7.3 system.
> 
> Packages are created by https://github.com/packpack/packpack.
> ```

ok

**Comment by ylobankov** [comment]
Cherry picked to 2.10.

**Pull request merged**